cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -133,6 +134,26 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
+  @Test def use_index_with_where_using_multiple_range_comparisons() {
+    // Need to make index preferable in terms of cost
+    executePreparationQueries((0 to 300).map { i =>
+      "CREATE (:Person)"
+    }.toList)
+    profileQuery(
+      title = "Use index with `WHERE` using multiple range comparisons",
```

Comment:
Reword title as per https://github.com/neo4j/neo4j-documentation/pull/241 for consistency